src/app/personside/visittkort-v2/header/navkontor/NavKontorContainer.tsx

```diff
@@ -14,9 +15,17 @@ const NavKontorSection = styled.section`
 `;
 
 function NavKontorContainer({ person }: Props) {
-    const navKontorInfo = person.navEnhet
-        ? `NAV-kontor / ${person.navEnhet.id} ${person.navEnhet.navn}`
-        : 'NAV-kontor / Ingen enhet';
+    let navKontorInfo: string;
+    if (!person.navEnhet) {
+        if (person.geografiskTilknytning) {
+            navKontorInfo = `Nav-kontor / ${mapUgyldigGT(person.geografiskTilknytning)}`;
+        } else {
+            navKontorInfo = 'NAV-kontor / Ingen enhet';
+        }
+    } else {
+        navKontorInfo = `NAV-kontor / ${person.navEnhet.id} ${person.navEnhet.navn}`;
+    }
```

Comment:
Vi trenger ikke denne nøstingen her; 

```suggestion
if (person.geografiskTilknyttning === null) {
  navKontorInfo = 'NAV-kontor / Ingen enhet'
} else if (person.navEnhet === null) {
  navKontorInfo = 'NAV-kontor / ${mapUgyldigGT(person.geografiskTilknytning)}'
} else  {
  navKontorInfo = 'NAV-kontor / ${person.navEnhet.id} ${person.navEnhet.navn}'
}
```